core/services/keeper/registry_synchronizer_sync.go

```diff
@@ -117,6 +109,11 @@ func (rs *RegistrySynchronizer) syncUpkeep(getter upkeepGetter, registry Registr
 	if err != nil {
 		return errors.Wrap(err, "failed to get upkeep config")
 	}
+
+	if upkeep.ExecuteGas <= 0 {
+		return errors.New("execute gas is zero")
```

Comment:
can you also add upkeep ID in the message?

Reply:
nit. can do an exact check with 0, since it's a uint32

Reply:
Done